test/blackbox-tests/test-cases/bigarray-select.t

```diff
@@ -0,0 +1,8 @@
+  $ echo "(lang dune 3.1)" > dune-project
+  $ cat >dune <<EOF
+  > (executable
+  >   (name c)
+  >   (libraries (select c.ml from (!bigarray -> c.nobigarray.ml) (bigarray -> c.bigarray.ml) ( -> c.dummy.ml))))
+  > EOF
+  $ touch c.bigarray.ml  c.dummy.ml  c.nobigarray.ml
+  $ dune build
```

Comment:
I’m not entirely sure what this test case is supposed to show. Unless I’m missing something, this would succeed before as well

Reply:
You're right, The test supposed to verify if `c.bigarray.ml` or `c.nobigarray.ml` appears in `_build/default/c`. If found then the test fail. I'm gonna fix it.

Reply:
I mean for OCaml 5.00 version of course.